src/scripts/productsTabs.js

```diff
@@ -0,0 +1,60 @@
+const furnitureCategoryTabs = document.querySelectorAll('.category-tab-select');
+
+furnitureCategoryTabs.forEach(function (tab) {
+  tab.addEventListener('click', checkResolutionOfScreen);
+});
+
+function checkResolutionOfScreen (event) {
+  const targetTab = event.currentTarget.getAttribute('href');
+  const tabPanelId = targetTab.substring(1, targetTab.length);
+  const tabPanel = document.getElementById(tabPanelId);
+  const tabPanelProductElements = tabPanel.children[0].children.length;
```

Comment:
nie lepiej uzyc tu jakiegos `querySelector`? w przypadku gdy html sie zmieni (np. dojdzie nowy wrapper) to kod przestanie działać

Reply:
Zamieniłem na `querySelector`